python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -798,13 +799,8 @@ def __get_app_store_proxy_setting(self):
 
         :returns: The http proxy connection string.
         """
-        try:
-            config_data = shotgun.get_associated_sg_config_data()
-        except UnresolvableCoreConfigurationError:
-            config_data = None
-
-        if config_data and constants.APP_STORE_HTTP_PROXY in config_data:
```

Comment:
This is where things got lost in translation regarding the missing vs empty I was mentioning above.